### PR TITLE
fix : bad variable

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

File: htdocs/fourn/commande/card.php [hunk 1]
```diff
@@ -1538,7 +1538,7 @@
 	print '<td>';
 	$usehourmin=0;
 	if (! empty($conf->global->SUPPLIER_ORDER_USE_HOUR_FOR_DELIVERY_DATE)) $usehourmin=1;
-	print $form->selectDate($datelivraison?$datelivraison:-1, 'liv_', $usehourmin, $usehourmin, '', "set");
+	print $form->selectDate($date_livraison?$date_livraison:-1, 'liv_', $usehourmin, $usehourmin, '', "set");
```
Comment (eldy) [inline htdocs/fourn/commande/card.php]: Variable $date_livraison is not defined where $datelivraison is.
Why this change ?

Comment (grandoc) [reply htdocs/fourn/commande/card.php]: within pdf muscadet if you make a var_dump($object); you get back property "date_livraison" and not datelivraison. And it works with $date_livraison...

Comment (grandoc) [reply htdocs/fourn/commande/card.php]: I have fixed it by a forward but the others commits are still waiting.

Comment (eldy) [reply htdocs/fourn/commande/card.php]: But here you are not into the code of pdf muscadet but into the page htdocs/fourn/commande/card.php
And in this page, variable $date_livraison is never defined generating a warning.
Please describe how to reproduce the bug. i can't reproduce on my environment.

**Comment by grandoc** [comment]
Laurent you closed with unmerged commits. All my work is in the trashcan ?
